pages/content/posts/events-restructure.md

```diff
@@ -0,0 +1,101 @@
+
+```js
+const btn = document.querySelector('button');
+btn.addEventListener('click', event => { // event is a MouseEvent
+  console.log(event.clientX)
+});
+```
+
+To summarize, events come with four different facets that need to be talked about in reference documentation one way or another:
+
+- Event names (types) for `addEventListener` and friends
+- Event handler properties (`on-*`)
+- HTML event handler content attributes
+- Event objects
+
+## Agreeing on content and compatibility data structures
+
+Now that we know the main different facets that make up events on the web platform, we took some time to assess how MDN documents events. A main focus was on de-duplicating reference pages, given that MDN's convention was to have at least two reference pages for each event: One for the event handler property (`on-*`) and a very similar page for the event name (types for usage in `addEventListener` and friends. We proposed to [unifying event and event handler pages](https://github.com/mdn/content/discussions/9098) instead.
+
+When unifying these two, it seemed like a good idea to also come up with a new page structure that represents both facets. In the discussion [event page type & structure ](https://github.com/mdn/content/discussions/10544), we agreed on how document events on a single page for each event going forward.
+
+More discussions happened on [mdn/browser-compat-data](https://github.com/mdn/browser-compat-data) where Philip Jägenstedt proposed to rethink how we represent events in compatibility data by also de-duplicating compat data entries and using `partial_support` annotations when one or more facets of an event is not supported.
+
+- [Representing support of events](https://github.com/mdn/browser-compat-data/issues/7545)
+- [OWD project: events](https://github.com/openwebdocs/project/issues/61)
+
+## Updating everything according to the new guidelines
+
+To test our new structures, we field tested and made updates to a few event pages on MDN and restructured the compatibility data accordingly. When we were happy, we investigated how much events there are on MDN and in BCD and came up with a [really long backlog](https://github.com/mdn/browser-compat-data/issues/14578) of events to go through. Basically, we signed ourselves up to rewrite all documented events of the web platform.
+
+Updating, moving, redirecting, deleting pages, and restructuring compatibility data so it all makes sense, isn't a trivial task. Thanks to the work of Jean-Yves Perrier, we were able to come up with a checklist that would guide to rewrite pages cleanly and consistently for all event pages.
```

Comment:
I think it'd be beneficial to omit "so it all makes sense". The part where you say "restructuring" has this implication. 
```suggestion
Updating, moving, redirecting, deleting pages, and restructuring compatibility data isn't a trivial task. Thanks to the work of Jean-Yves Perrier, we were able to come up with a checklist that would guide to rewrite pages cleanly and consistently for all event pages.
```